puzzlegenerator/puzzlegenerator_ui/src/main/java/com/flurnamenpuzzle/generator/Main.java

```diff
@@ -12,10 +13,19 @@
 public class Main {
 
 	public static void main(String[] args) {
+		setSystemLookAndFeel();
 		PuzzleGeneratorModel puzzleGeneratorModel = new PuzzleGeneratorModel();
 		PuzzleGeneratorController puzzleGeneratorController = new PuzzleGeneratorController(
 				puzzleGeneratorModel);
 		SwingUtilities.invokeLater(puzzleGeneratorController::initializeView);
 	}
 
+	
+	public static void setSystemLookAndFeel(){
+		try {
+			// Set System L&F

```

Comment:
Den Kommentar braucht es nicht ;-)
Allerdings könntest du UIManager.getSystemLookAndFeelClassName() in eine eigene Variable auslagern, das macht den Code lesbarer